packages/longhorn/longhorn-1.4/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
 commit: 82be2970c8ffb8b50d6fcad77a3326d8e2434012
-version: 101.2.0
+version: 102.2.0
```

Comment:
This should be 102.0.0

Reply:
@PhanLe1010 1.2 is EOL. I think we can remove it from the release.

cc @rebeccazzzz

Reply:
Right, I forgot about Longhorn's multiple release lines. It makes sense now.

Reply:
@innobead I see. Is it ok to remove it in the next major release bump (to 103.x.x) just in case we need to add a hot fix in Longhorn 1.2.x? If not, I will remove the 1.2.x release line and make the versions:
* 102.0.0 for Longhorn 1.3.x
* 102.1.0 for Longhorn 1.4.x

Reply:
Yeah, we don't really have a clear process about when to discontinue the chart release if it reaches EOL. Probably we can do this when there is a major version (Rancher chart) bump. 

Let's start from 103 and keep 1.2.x as is first to have some buffer for any exception. However, we should not have any hotfix for 1.2.x any more. 

cc @rebeccazzzz to understand this new policy. 

@PhanLe1010 Could you help document this on confluence to mention we will review if remove the old release chart when a new major version bump of Rancher chart? Thanks.

Reply:
@innobead Updated the doc https://confluence.suse.com/display/LON/How+to+create+Rancher+Chart+PRs
Thank you